hubmap/frontend/cypress/integration-examples/examples/cypress_api.spec.js

```diff
@@ -114,7 +114,7 @@ context('Cypress.config()', () => {
     expect(myConfig).to.have.property('baseUrl', null)
     expect(myConfig).to.have.property('defaultCommandTimeout', 4000)
     expect(myConfig).to.have.property('requestTimeout', 5000)
-    expect(myConfig).to.have.property('responseTimeout', 30000)
+    expect(myConfig).to.have.property('responseTimeout', 60000)
```

Comment:
This is just example code... It shouldn't need to change? (When the example code should even be checked in is a valid question... I think it's a helpful resource, but if it's distracting, we can get rid of it.)

Reply:
Well, when redirecting, ideally it is **not** recommended to test <a> tags/redirects outside of domain in cypress. //https://docs.cypress.io/guides/references/error-messages.html#Cypress-detected-a-cross-origin-error-happened-on-page-load
 However, since you did not establish what you preferred in your original issue, I have implemented both. So using redirects, it takes a little more time, depending on network. so I increased it as per Cypress forums discussion.

Reply:
I'm still confused: This example code shouldn't even be running. If this change had any effect, something is wrong.